test/tiles_rendered.js

```diff
@@ -1,8 +1,30 @@
-const testTile = function (prefix, z, x, y, format, status, scale, type) {
+var testTile = function (
+  prefix,
+  tileSize = 256,
+  z,
+  x,
+  y,
+  format,
+  status,
+  scale,
+  type,
+) {
   if (scale) y += '@' + scale + 'x';
-  const path = '/styles/' + prefix + '/' + z + '/' + x + '/' + y + '.' + format;
+  var path =
```

Comment:
this is fine as-is, if you wanted to you could use `require('path')` to join it all together:
```js
path.join('/styles', prefix, tileSize, z, x, `${y}.${format}`);
```